string class_name
float32 probability
int16 x_min
int16 y_min
int16 x_max
int16 y_max